contracts/FeeManager/FeeManager.sol

```diff
@@ -102,4 +102,62 @@ contract FeeManager is IFeeManager {
     ) external view override returns (uint256) {
         return _balances[partner];
     }
+
+    function getPool() public view returns (address) {
+        return _pool;
+    }
+
+    function setPool(address newPoolAddress) public onlyAuthorised {
```

Comment:
the modifer "onlyAuthrorised" checks if the call has been made from one of the whitelisted registrar/renewer contracts. this operation is expected to be performed by the admin of the protocol